backend/api.py

```diff
@@ -0,0 +1,86 @@
+from flask import Flask, request, jsonify
+from flask_cors import CORS
+import os
+from copy import copy
+from backend.position import Position
+from backend.piece import Piece
+from backend.player import Player
+from backend.board import Board
+from backend.turn import Turn
+from backend.picked import Picked
+template_folder_path = os.path.abspath('./frontend/src')
+app = Flask(__name__, template_folder=template_folder_path)
+CORS(app)
+
+
+board = Board()
+board.create_players()
+chosen_piece = Picked()
+turn = Turn(player="red")
+output_json = {}
+
+rows = {}
+@app.route("/", methods=['GET'])
+def board_output():
+
+    for row in range(8):
+        rows["row"+str(row)] = board.get_line_row(row)
+    output_json["board"] = rows
+    output_json["turn"] = turn.get_player()
+    return jsonify(output_json)
+
+
+@app.route('/restart', methods=['GET'])
+def restart():
+    board.reset()
+    board.create_players()
+    turn.set_player("red")
+    return "OK"
+
+
+@app.route('/pick', methods=['POST'])
```

Comment:
this whole method looks bad, improve the logic of the 'ifs' and use more methods to explain whats going on in that section